go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/istio-ecosystem/sail-operator
 
-go 1.24.0
+go 1.23.0
```

Comment:
Had to downgrade go to `1.23.0` to match the istio version for 1.25: https://github.com/istio/istio/blob/9dba3490d112cca67ffbd9ccb4c0ef8c65211f25/go.mod#L3